boards/arm/rcar_h3ulcb/rcar_h3ulcb_cr7_defconfig

```diff
@@ -9,3 +9,5 @@ CONFIG_CONSOLE=y
 CONFIG_RAM_CONSOLE=y
 CONFIG_FLASH_SIZE=0
 CONFIG_FLASH_BASE_ADDRESS=0
+CONFIG_GPIO=y
+CONFIG_GPIO_RCAR=y
```

Comment:
GPIO_RCAR could be moved to an soc kconfig file so that it doesn't have to be defined in every board.

Reply:
about this one, I think prefer to keep it at board level. Since I/O are shared between the multiple processors on this platform,
it let the board integrator the choice to enable or not this driver.

Reply:
What I meant by this was that GPIO_RCAR could move to the soc kconfig file, but CONFIG_GPIO would still be enabled at the board level, i.e. it would work like the clock control kconfig from the other change I mentioned.

Reply:
got it! thanks a lot

Reply:
hum, perhaps not, let's imagine that you want to drive an I2c gpio expander, then you don't necessary need GPIO_RCAR.

Reply:
fair enough